lightspeed/modules/proc_generate-playbook.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: PROCEDURE
+
+[id="generate-playbook_{context}"]
+
+= Generating Ansible playbooks
+
+You can use the natural language interface in the Ansible VS Code extension to generate an entire Ansible playbook.
+
+.Prerequisites
+
+* You are part of an organization that has a trial or paid subscription to {PlatformName} and {ibmwatsonxcodeassistant} both.
+* You have installed and configured the Ansible VS Code extension. 
+
+.Procedure
+
+. Log in to VS Code with your Red Hat account.
```

Comment:
Is it VS Code or VSCode?

Reply:
It is VS Code. See https://code.visualstudio.com/.